views/sponsors.php

```diff
@@ -27,6 +29,124 @@
     <!-- Navbar -->
     <?php include('../views/includes/navbar.html'); ?>
 
+    <!--Body -->
+    <div class = "allcontent">
+ <h1 align = "center" style = "margin-top: 50px">
+  <span class="black">OUR</span>
```

Comment:
Since the default font-color is black, you don't need to wrap the "OUR" part with a span tag.